waku/v2/protocol/filter/client.go

```diff
@@ -44,8 +45,8 @@ type WakuFilterLightNode struct {
 }
 
 type ContentFilter struct {
-	Topic         string
-	ContentTopics []string
+	PubsubTopic   string
+	ContentTopics ContentTopicSet
```

Comment:
I don't think we should call this a Set as no sorted order is maintained. Wouldn't it be better to use string array itself?

Reply:
@chaitanyaprem thank you for reviewing! With this naming, i wanted to avoid potential duplication issues. The `Set` name is used to signal that 
a) it should only contains unique elements
b) order is not important, as you pointed out.
This als seems to be a common pattern across go-waku code, we often use similar structures, e.g. also `PeerSet` in `subscribers_map.go`, etc.

Reply:
Got it, also heard your reasoning for keeping it in tandem with status-go data structures.